pyaedt/modeler/stackup_3d.py

```diff
@@ -0,0 +1,2284 @@
+        created_line = Line(
+            self._app,
+            frequency,
+            line_width if is_impedance else None,
+            line_width if not is_impedance else None,
+            self,
+            dielectric_layer,
+            line_length=line_length if not is_electrical_length else None,
+            line_electrical_length=line_length if is_electrical_length else None,
+            line_position_x=line_position_x,
+            line_position_y=line_position_y,
+            line_name=line_name,
+            reference_system=reference_system,
+            axis=axis,
+        )
+        created_line.aedt_object.group_name = "Layer_{}".format(self._name)
+        self._obj_3d.append(created_line.aedt_object)
+        self._stackup._object_list.append(created_line)
+        return created_line
+
+    @pyaedt_function_handler()
+    def polygon(self, points, material="copper", is_void=False, poly_name=None):
+        """Create a new Polygon.
+
+        Parameters
+        ----------
+        points : List
+            Points list of [x,y] coordinates.
+        material : str, optional
+            Material name.
+        is_void : bool, optional
+            Either if the polygon is a void or not.
+            On ground layers it will act opposite as the boolean value since ground is negative.
+        poly_name : str, optional
+            Polygon Name.
+
+        Returns
+        -------
+
+        """
+        if not poly_name:
+            poly_name = generate_unique_name("{0}_poly".format(self._name), n=3)
+        polygon = Polygon(
+            self._app,
+            points,
+            thickness=self._thickness,
+            signal_layer_name=self._name,
+            mat_name=material,
+            is_void=is_void,
+            poly_name=poly_name,
+        )
+        polygon.aedt_object.group_name = "Layer_{}".format(self._name)
+
+        if self._layer_type == "ground":
+            if not is_void:
+                if polygon.aedt_object.is3d:
+                    self._app.modeler[self._name].subtract(polygon.aedt_object, True)
+                    polygon.aedt_object.material_name = self.filling_material_name
+                else:
+                    self._app.modeler[self._name].subtract(polygon.aedt_object, False)
+                    return True
+        elif is_void:
+            if polygon.aedt_object.is3d:
+                self._app.modeler.subtract(self._obj_3d, polygon.aedt_object, True)
+                polygon.aedt_object.material_name = self.filling_material_name
+            else:
+                self._app.modeler[self._name].subtract(polygon.aedt_object, False)
+                return True
+        else:
+            self._app.modeler.subtract(self._obj_3d[0], polygon.aedt_object, True)
+            self._obj_3d.append(polygon.aedt_object)
+            self._stackup._object_list.append(polygon)
+        return polygon
+
+
+class PadstackLayer(object):
+    """PadstackLayer Class member of Padstack."""
```

Comment:
The description for the class should begin with a verb with an 's', such as "Provides ...." so that the summary table is formatted correctly.